middlewares/validators/user.js

```diff
@@ -106,6 +107,9 @@ async function getUsers(req, res, next) {
           "string.empty": "page must contain a positive number or zero",
           "string.pattern.base": "page value either be a positive number or zero",
         }),
+      search: joi.string().optional().messages({
```

Comment:
is it a part of this PR? why do we need this?

Reply:
Resolved